### ISSUE TITLE
Passwords should be entered with console "echo"

### ISSUE THREAD

**Issue by tenthirtyam** [vmware/powershell-module-for-vmware-cloud-foundation-power-management#55]
### Code of Conduct

- [X] I have read and agree to the project's Code of Conduct.
- [X] Vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue initial description to help the maintainers prioritize.
- [X] Do not leave "+1" or other comments that do not add relevant information or questions.
- [X] If you are interested in working on this issue or have submitted a pull request, please leave a comment.

### Description

Passwords for connecting to different components are shown when prompted. 

### Use Case(s)

Password prompts should hide the string that is entered. For example, `-AsSecureString` as one possible solution.

### Potential Configuration

We should enhance this:

1. in the scripts itself (when the password is not provided as parameter).
2. In the module - if we run some function from the module without parameters the user will be asked for input parameters, one of those parameters will be a password prompt.

Basically, in all places that we are asking for password as input parameter, these should be masked.

**Comment by tenthirtyam** [comment]
Resolved in https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-power-management/pull/73.
